src/strategies/erc721/schema.json

```diff
@@ -10,7 +10,6 @@
           "type": "string",
           "title": "Symbol",
           "examples": ["e.g. DOODLE"],
-          "minLength": 1,
```

Comment:
Shouldn't symbol also be removed inside required?
<img width="957" alt="image" src="https://user-images.githubusercontent.com/51686767/156765807-2b8bc735-84ce-4cf1-b239-e612b2b9c77f.png">

Reply:
Removed 👍